package.json

```diff
@@ -39,7 +39,7 @@
   },
   "homepage": "https://github.com/theia-ide/theia-trace-extension",
   "devDependencies": {
-    "@theia/cli": "1.21.0",
+    "@theia/cli": "1.29.0",
     "jsonc-parser": "^3.0.0",
     "lerna": "^4.0.0",
     "typescript": "4.5.4"
```

Comment:
Theia's own latest TS version says "~4.5.5"; amend here too?

Reply:
One more thing: #690 and the earlier #461 had more updates upon bumping Theia's versions that way. Does this PR also require similar dependency upgrades, say, based on Theia's corresponding bumps (to align)?

Reply:
Yes, I think it's a good idea to keep the TS dependency on par with Theia. Upgraded.
Thank you for bringing up those other PRs. I upgraded all the other related dependencies.